admin/config/locales/en.yml

```diff
@@ -3849,6 +3855,37 @@ en:
           service: Service
           service_code: Service Code
           tax_code: Tax Code
+      shipping_skus:
+        unit:
+          inches:
+            one: '%{count}in'
+            other: '%{count}in'
+          centimeters:
+            one: '%{count}cm'
+            other: '%{count}cm'
+          ounces:
+            one: '%{count}oz'
+            other: '%{count}oz'
+          grams:
+            one: '%{count}g'
+            other: '%{count}g'
+        edit:
+          title: Attributes for %{name}
+        index:
+          button: Add New Shipping SKU
+          title: Shipping
+          sku_pluralize:
+            one: "%{count} SKU"
+            other: "%{count} SKUs"
+        new:
+          create_sku: Create Shipping SKU
+          title: New Shipping SKU
+        saved: "%{sku} has been saved."
```

Comment:
Conventionally, we put these in a `flash_messages` section.

Reply:
i just copied this from the pricing skus controller, can change it if you want though. https://github.com/workarea-commerce/workarea/blob/master/admin/app/controllers/workarea/admin/pricing_skus_controller.rb#L43-L44